server/lib/util.js

```diff
@@ -95,3 +96,23 @@ function resolveObject(object) {
       return settled;
     });
 }
+
+/**
+ * @method dateFormatter
+ *
+ * @description
+ * Accepts an object of key/value pairs. Returns the same object with all values
+ * that are dates converted to a standard format.
+ */
+function dateFormatter(Rows, dateFormat) {
```

Comment:
It is generally good practice to only use capital letters with constructors - functions that you will call `new` on to invoke a new instance.  We occasionally break this rule, but it isn't necessary to break it here.  `Rows` is just a flat array.

Reply:
This shouldn't be capitalized.